base/src/main/kotlin/net/corda/v5/base/types/LayeredPropertyMap.kt

```diff
@@ -7,57 +7,90 @@ package net.corda.v5.base.types
  * The layered property map provides simple conversions to a possibly complex objects which can use several keys in
  * dot-notation. Take as an example of the map:
  *
- * "corda.name" to "CN=me, O=R3, L=Dublin, C=Ireland",
- * "corda.sessionKey" to "ABCDEF...",
- * "corda.endpoints.0.url" to "localhost",
- * "corda.endpoints.0.protocolVersion" to "1",
- * "corda.endpoints.1.url" to "localhost",
- * "corda.endpoints.1.protocolVersion" to "2"
+ * - "corda.name" to "CN=me, O=R3, L=Dublin, C=Ireland",
```

Comment:
C attribute should be a two-letter country code, in this case "IE".

Reply:
Nice catch!